plugin/pkg/scheduler/algorithm/priorities/taint_toleration_test.go

```diff
@@ -204,6 +204,18 @@ func TestTaintAndToleration(t *testing.T) {
 				{Host: "nodeC", Score: 0},
 			},
 		},
+		{
+			test: "Default behaviour No taints and tolerations, lands on node with no taints",
+			//pod without tolerations
+			pod: podWithTolerations([]v1.Toleration{}),
+			nodes: []*v1.Node{
+				//Node without taints
```

Comment:
I think add one more node with some taints. And then test to see if the pod lands on the node without taints. So just adding one more node with taints should do.